docs/output.md

```diff
@@ -435,23 +447,17 @@ For a Tumor/Normal pair only:
 - `[TUMORSAMPLE].pileup.gz_BAF.txt` and `[NORMALSAMPLE].pileup.gz_BAF.txt`
   - file with beta allele frequencies for each possibly heterozygous SNP position
 
+For further reading and documentation see the [Control-FREEC manual](http://boevalab.com/FREEC/tutorial.html).
+
 ### MSI status
 
-[Microsatellite instability](https://en.wikipedia.org/wiki/Microsatellite_instability)
-is a genetic condition associated to deficiencies in the
-mismatch repair (MMR) system which causes a tendency to accumulate a high
-number of mutations (SNVs and indels).
+[Microsatellite instability](https://en.wikipedia.org/wiki/Microsatellite_instability) is a genetic condition associated to deficiencies in the mismatch repair (MMR) system which causes a tendency to accumulate a high number of mutations (SNVs and indels).
+An altered distribution of microsatellite length is associated to a missed replication slippage which would be corrected under normal MMR conditions.
 
 #### MSIsensor
 
-[MSIsensor](https://github.com/ding-lab/msisensor) is a tool to detect the MSI
-status of a tumor scanning the length of the microsatellite regions. An altered
-distribution of microsatellite length is associated to a missed replication
-slippage which would be corrected under normal mismatch repair (MMR) conditions. It requires
-a normal sample for each tumour to differentiate the somatic and germline
-cases.
-
-For further reading see the [MSIsensor paper](https://www.ncbi.nlm.nih.gov/pubmed/24371154).
+[MSIsensor](https://github.com/ding-lab/msisensor) is a tool to detect the MSI status of a tumor scanning the length of the microsatellite regions.
+It requires a normal sample for each tumour to differentiate the somatic and germline cases.
 
 For a Tumor/Normal pair only:
 **Output directory: `results/VariantCalling/[TUMORSAMPLE]_vs_[NORMALSAMPLE]/MSIsensor`**
```

Comment:
Not sure what you meant there

Reply:
Ah sorry, yes it is the paragraph below. There are `.` at the of the bullet points even though it is not a sentence, so figured it could be removed. Should have added a comment here